Lib/Inc/error_log.h

```diff
@@ -0,0 +1,40 @@
+/**
+ * @file error_log.h
+ */
+
+/*
+ *  Created on: Feb 12, 2022
+ *      Author: niuslar
+ */
+
+#ifndef ERROR_LOG_H_
+#define ERROR_LOG_H_
+
+#include <string>
+#include "uart_com.h"
+
+class CLog
```

Comment:
Is this a logging class or error logging class? Perhaps more verbose name would help. Generally, the rule is that class name and file name should match.